src/js/about-gallery.js

```diff
@@ -1,17 +1,20 @@
 export function initGallery() {
   const galleryImages = Array.from(document.querySelectorAll('.gallery__image'));
+  let activeImage = galleryImages.find(image => image.classList.contains('gallery__image--active'));
 
   galleryImages.forEach(image => {
     image.addEventListener('mouseover', e => {
-      activateGalleryImage(e.target);
-
-      galleryImages
-        .filter(image => image !== e.target)
-        .forEach(image => deactivateGalleryImage(image));
+      toggleGalleryImage(activeImage, e.target)
```

Comment:
Could you add `const nextImage = e.target;` here and not use `e.target` below?
Thanks!